conf/giga.config

```diff
@@ -0,0 +1,106 @@
+/*
+ * GIGA CLUSTER
+ * ============
+ * This configuration file provides sensible defaults to run Nextflow pipelines
+ * on the GIGA cluster.
+ *
+ * For more information on the GIGA cluster, refer to this page of the wiki:
+ *     https://giga-bioinfo.gitlabpages.uliege.be/docs/mass-storage-and-cluster/cluster/overview.html
+ * If you face any issue using this configuration, please contact the GIGA
+ * Bioinformatics Team:
+ *     https://giga-bioinfo.gitlabpages.uliege.be/docs/mass-storage-and-cluster/about/contact.html#bioinformatics-team
+ */
+params {
+    config_profile_name        = 'GIGA'
+    config_profile_description = 'GIGA cluster profile provided by the GIGA Bioinformatics Team.'
+    config_profile_contact     = 'Martin Grignard (@MartinGrignard)'
+    config_profile_url         = 'https://giga-bioinfo.gitlabpages.uliege.be/docs/mass-storage-and-cluster/cluster/overview.html'
+}
+
+/*
+ * Resources limitations
+ * ---------------------
+ * These resources limitations are maximum values across all nodes of all
+ * queues. At least one node matches these maximum resources in all of the
+ * available queues.
+ *
+ * For more information on the available nodes on the GIGA cluster, refer to
+ * this page of the wiki:
+ *     https://giga-bioinfo.gitlabpages.uliege.be/docs/mass-storage-and-cluster/cluster/overview.html#nodes
+ * or use the `sinfo` command.
+ */
+params {
+    max_cpus   = 48
+    max_memory = 512.GB
+    max_time   = 365.days  // Virtually infinite (jobs without a time limit run in the 'kosmos' partition).
+}
+
+/*
+ * Singularity configuration
+ * -------------------------
+ * Singularity is used to run containerised tools. On the cluster, one should
+ * use the following command to make it available:
+ *     $ module load singularity
+ * This configuration makes sure to cache the downloaded containers into a
+ * shared directory on the resources SSDs.
+ *
+ * For more information on how to use Singularity on the GIGA cluster, refer to
+ * this page of the wiki:
+ *     https://giga-bioinfo.gitlabpages.uliege.be/docs/mass-storage-and-cluster/tools/environment/singularity.html
+ */
+singularity {
+    autoMounts  = true
+    cacheDir    = '/resources/GIGA/PUBLIC/tools/.cache/singularity'
```

Comment:
Just to confirm this is definitely available for all users?

Reply:
Yes, the `/resources/GIGA/PUBLIC/tools/.cache/singularity` path is located on a shared storage and has read and write permissions for all the people who have access to the GIGA cluster.